.golangci.yml

```diff
@@ -4,6 +4,9 @@ linters-settings:
   cyclop:
     max-complexity: 15
     skip-tests: true
+  dupl:
+    # tokens count to trigger issue, 150 by default
+    threshold: 350
```

Comment:
scaled upwards in jumps of 50 so threshold at 300 was failing for me

Reply:
yeah, we have duplication in tests. This is fine for now, until we can write some common test code to dedup.

Reply:
From [Go Proverbs](https://go-proverbs.github.io/): _A little copying is better than a little dependency_.